conda/cli/common.py

```diff
@@ -405,7 +405,7 @@ def name_prefix(prefix):
     return basename(prefix)
 
 def check_write(command, prefix, json=False):
-    if inroot_notwritable(prefix):
+    if inroot_notwritable(prefix) or prefix == '/usr':
```

Comment:
I was looking to prevent installs into /usr overwriting system libraries.  I think my preferred mechanism at this point is to not create /usr/conda-meta, so this could go but would otherwise just be a safety check.  I had been creating /usr/conda-meta so that basic conda commands ran without an immediate error.